chemprop/data/datasets.py

```diff
@@ -355,9 +367,14 @@ def normalize_targets(self, scaler: StandardScaler | None = None) -> StandardSca
         return self.datasets[0].normalize_targets(scaler)
 
     def normalize_inputs(
-        self, key: str | None = "X_d", scaler: StandardScaler | None = None
+        self, key: str | None = "X_d", scaler: list[StandardScaler] | None = None
```

Comment:
`dset.normalize_inputs(key=None)` returns a list of scalers that loops through each of `"X_d", "V_f", "E_f", and "V_d"`. I would almost remove None as an option for a monocomponent dataset because it changes the return type of normalize_inputs, but I can see some convenience value. I don't think it makes sense for multicomponent normalize_inputs to accept key=None.
```suggestion
        self, key: str = "X_d", scaler: list[StandardScaler] | None = None
```